src/solver/conv_asm_1x1u_stride2.cpp

```diff
@@ -99,6 +99,14 @@ inline static bool IsLinear(const int v)
     return L <= v && v <= H;
 }
 
+static inline size_t divide_round_plus_inf(const size_t x, const unsigned y)
+{
+    assert(x >= 0 && y > 0);
```

Comment:
@atamazov  like mentioned in the other PR, CI has some HIP tidy issue with this line:
```
[2023-01-11T01:44:14.744Z] /home/jenkins/workspace/MLLibs_MIOpen_PR-1898/src/solver/conv_asm_1x1u_stride2.cpp:104:14: warning: Unsigned expression 'x' can't be negative so it is unnecessary to test it. [unsignedPositive]
[2023-01-11T01:44:14.744Z]     assert(x >= 0 && y > 0);
[2023-01-11T01:44:14.744Z]              ^

Reply:
This assert is not necessary here. Leftover after moving from generic_search.hpp.